src/Amazon.AspNetCore.Identity.Cognito/Amazon.AspNetCore.Identity.Cognito.csproj

```diff
@@ -40,9 +40,9 @@
 
   <ItemGroup>
     <PackageReference Include="Amazon.Extensions.CognitoAuthentication" Version="3.0.0-preview.2" />
```

Comment:
This is going to get bumped to preview 3. I'd wait to get the other library out and then upgrade this to preview 3 while you're at it.